src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -40,6 +40,7 @@
       "type": "build",
       "version": "1.1.0-*"
     },
+    "Microsoft.Extensions.Logging": "1.1.0-preview2-22526",
```

Comment:
Thank you for the help! I had to suppress the CS0618 warnings in the Properties for the Core.Test project to compile after adding the Obsolete attribute. Is that acceptable?